PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -294,6 +294,21 @@ Although not recommended, if for some reason a user needs to be deleted, follow
 
 To delete the username, click on the OK button. To cancel and keep the username, click on the CANCEL button. This action cannot be undone so proceed with caution.
 
+## Configuring Manager Dashboard
+
+The modules that show up on the manager are cofigurable from the roles settings. To change the dashboard for a for a role, click on the **Settings** tab and click on **Roles**. Scroll down to the **Dashboards** section and here you can add or remove modules.
+
+The new modules for version 1.4 are as follows:
+
+* View Approvals
+* View Assessment
+* View Cases To Assign
+* View None Values
+
+## Referral Button
+
+Version 1.4 adds a new referral button to a Response Overview. To have the button appear, the user has to have the permission **Refer Case from Service form** added to their role (this is disabled by default)
```

Comment:
Once again, don't talk about "v1.4 adds." Just say that you can have a referral button on the Response Overview form, what it does, and what in the configuration needs to be changed to have this button appear.